change/@fluentui-web-components-085483f0-a46f-4531-b8c4-218a2e78461b.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
```

```suggestion
  "type": "none",
```

As this PR does not affect published package, mark it as "none"